python/sanlock.c

```diff
@@ -1674,33 +1676,28 @@ initexception(void)
     return excp;
 }
 
-PyMODINIT_FUNC
-initsanlock(void)
+static int
+__module_init(PyObject* m)
```

Comment:
No more underscore prefix please.

Reply:
suggesting the "util_" prefix for internal functions.